### ISSUE TITLE
mmCIF parser file handler

### ISSUE THREAD

**Issue by sunhwan** [biopython/biopython#536]
mmCIF parser currently only accept filename as an argument. The MMCIF2Dict.py should be edited as below to be consistent with other parsers.

Current:

```
from Bio._py3k import input as _input

import shlex

class MMCIF2Dict(dict):

    def __init__(self, filename):
        with open(filename) as handle:
            loop_flag = False
```

Edit: 

```
from Bio.File import as_handle
from Bio._py3k import input as _input

import shlex

class MMCIF2Dict(dict):

    def __init__(self, file):
        with as_handle(file) as handle:
            loop_flag = False
```

**Comment by peterjc** [comment]
That looks good, except ``file`` is a reserved word in Python so better not used as the argument name. I would leave it as ``filename``, but add a docstring to explain this can be a handle.

(The whole module would really benefit from some proper docstring documentation)

Could you do this as a pull request please?

**Comment by peterjc** [comment]
The pull request #537 looks good, other than needing a docstring. Thank you.

If you wanted to go further, it would be great to add unit tests for ``MMCIF2Dict`` in ``test_MMCIF.py`` (using a handle and using a filename).

**Comment by speleo3** [comment]
Looks like this ticket can be closed (should have been closed with the merge of #537).

**Comment by peterjc** [comment]
Thanks!
